libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -563,6 +563,16 @@ singleton AP_Winch method release_length void float'skip_check
 singleton AP_Winch method set_desired_rate void float'skip_check
 singleton AP_Winch method get_rate_max float
 
+include AP_Parachute/AP_Parachute.h depends APM_BUILD_TYPE(APM_BUILD_ArduPlane)||APM_BUILD_COPTER_OR_HELI
+singleton AP_Parachute depends APM_BUILD_TYPE(APM_BUILD_ArduPlane)||APM_BUILD_COPTER_OR_HELI
+singleton AP_Parachute rename parachute
+singleton AP_Parachute method enabled boolean
+singleton AP_Parachute method release void
+singleton AP_Parachute method released boolean
+singleton AP_Parachute method release_initiated boolean
+singleton AP_Parachute method release_in_progress boolean
```

Comment:
What is the use case here? Would we be better to return a enum state with a single method?

Reply:
Agree. However, in this case, we should rewrite the AP_Parachute module to implement that method right?

Reply:
Yeah you would have to add the enum first.

Reply:
Done with the enum. Any clues on how to define it on the bindings file? Haven't found the way yet

Reply:
@IamPete1 , can you give me any clue on this please? Is it documented somewhere?